web/beacon-app/src/features/onboarding/utils.ts

```diff
@@ -0,0 +1,30 @@
+import { t } from '@lingui/macro';
+export const getWorksOptions = () => {
+  return [
+    { value: 'Application Development', label: t`Application Development` },
+    { value: 'Data science', label: t`Data science` },
```

Comment:
Should `Data engineering` be added here?
https://www.figma.com/file/PvL23fm6W36Zd88O1XVyIo/Beacon-DX-UI?type=design&node-id=8882-2714&mode=design&t=KYg1G5GshP8AWwwr-4